src/components/feature-board/content-elements/FileContentElementChips.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+	<div class="ma-2">
+		<v-chip class="mr-2" small>{{ fileExtension }}</v-chip>
+		<v-chip class="mr-2" small>{{ humanReadableFileSize }}</v-chip>
+	</div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent } from "vue";
+import {
+	convertFileSizeToHumanReadable,
+	getFileExtension,
+} from "@/utils/fileHelper";
+
+export default defineComponent({
+	name: "FileContentElementChips",
+	props: {
+		fileSize: {
+			type: Number,
+			required: true,
+		},
+		fileName: {
+			type: String,
+			required: true,
+		},
+	},
+
+	setup(props) {
+		const humanReadableFileSize = computed(() =>
+			convertFileSizeToHumanReadable(props.fileSize)
+		);
+
+		const fileExtension = computed(() =>
+			getFileExtension(props.fileName).toUpperCase()
+		);
+
+		return {
+			humanReadableFileSize,
+			fileExtension,
+		};
+	},
+});
+</script>
```

Comment:
I also found, that the `v-chips` are still having an `hover` effect. Please apply the changes suggested by @SevenWaysDP .

```suggestion
</script>

<style lang="scss" scoped>
.v-chip {
	opacity: 1;
}
</style>

```